test-support/helpers/ember-tooltips.js

```diff
@@ -101,9 +105,9 @@ function validateSide(side, testHelper = 'assertTooltipSide') {
 }
 
 function getTooltipAndTargetPosition(options = {}) {
-  const $target = getTooltipTargetFromBody(options.targetSelector);
+  const $target = getTooltipTargetFromBody(options.targetSelector || tooltipOrPopoverTargetSelector);
```

Comment:
This param was unintentionally passing in `null` as the selector causing elements to be incorrectly selected.